js/script.js

```diff
@@ -0,0 +1,70 @@
+$(document).ready(function(){
+
+  //questions and possible ingredients with flavor id which is currently unused
+  var currentQuestion = -1;
+
+  var questions = [
```

Comment:
As I understand it I would still need to write out all of those named values so it seems as though the  method I used would actually be less lines of code, however,  I will make an alternate version as I was unclear on that requirement.